_posts/2022-03-15-kubeflow-1.5-release.md

```diff
@@ -30,7 +30,7 @@ Note on K8s 1.22: Although many of the Kubeflow services (KFP, AutoML, Training
 
 ## Streamlined user experience
 
-In v1.5, the Kubeflow User Interface (UI) dashboards for Notebooks, AutoML and Volume Manager more closely match KFP’s dashboard.   These enhancements include how dashboard fields are displayed and how operations are performed.  This gives users a consistent look and feel when working in the UI.   Additionally, Kubeflow notebooks and the volume manager have implemented a new configuration template for configuring and using persistent volumes and Kubernetes storage classes.  This template makes it easier to expose more storage classes and will enable easier support of popular storage offerings, including NFS.  In addition, AutoML has also updated its SDK, CI framework and parameter settings across frameworks (goptuna, optuna, hyperopt).
+In v1.5, the Kubeflow User Interface (UI) dashboards for Notebooks, Tensorboards and Volume Manager more closely match KFP’s dashboard.   These enhancements include how dashboard fields are displayed and how operations are performed.  This gives users a consistent look and feel when working in the UI.   Additionally, Kubeflow notebooks and the volume manager have implemented a new configuration template for configuring and using persistent volumes and Kubernetes storage classes.  This template makes it easier to expose more storage classes and will enable easier support of popular storage offerings, including NFS.  In addition, Tensorboard has also updated its SDK, CI framework and parameter settings across frameworks (goptuna, optuna, hyperopt).
```

Comment:
Second  `Tensorboard` replacement-> Is it right?

Reply:
@johnugeorge good catch.   I believe that was AutoML and it was changed in the last update.    @shannonbradshaw would you pleases change Tensorboard has also updated its SDK, CI... to AutoML has also changed its SDK, CI...

Reply:
Thanks. Fixed. @johnugeorge please confirm?

Reply:
Looks good!